stylesheets/components/avatars/avatars.css

```diff
@@ -0,0 +1,48 @@
+@import "/stylesheets/index.css";
+
+.avatar-sm {
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+}
+
+.avatar-md {
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+}
+
+.avatar-lg {
+    width: 70px;
+    height: 70px;
```

Comment:
We should rem instead of px. It helps to make the website responsive.

Reply:
Thank you, Rahul! I wasn't sure if the avatars had to be made responsive. I'll make those changes.